package.json

```diff
@@ -138,17 +133,5 @@
   "license": "MIT",
   "lint-staged": {
     "*.{ts,tsx}": "eslint --cache --fix"
-  },
-  "eslintConfig": {
-    "overrides": [
-      {
-        "files": [
-          "**/*.stories.*"
-        ],
-        "rules": {
-          "import/no-anonymous-default-export": "off"
```

Comment:
unused

Reply:
Good remove!

Reply:
oh it must have been for the storybook stories. if it causes problems just re-add 👍

Reply:
Yes, will do!